system-variables.md

```diff
@@ -79,6 +79,45 @@ mysql> SELECT * FROM t1;
 - Default value: ON
 - Controls whether statements should automatically commit when not in an explicit transaction. See [Transaction Overview](/transaction-overview.md#autocommit) for more information.
 
+<<<<<<< HEAD
```

Comment:
@dveeden Could you help resolve the conflicts in the release branch?
@morgo Could you resolve the conflicts in https://github.com/pingcap/docs/pull/5793? Otherwise, there can be more conflicts every time any changes made in `system-variables.md` are cherry-picked from master to release-5.0. Thanks!

Reply:
Trying to do that now